spec/features/02_user_visits_trucks_index_spec.rb

```diff
@@ -17,5 +17,6 @@
     expect(page).to have_content('All Trucks')
     expect(page).to have_content(food_truck.name)
     expect(page).to have_content(food_truck.description)
```

Comment:
It looks like the test suite is failing here, because the food truck description isn't ON the index page. It looks like the location may not be, either. Please either update your ERB file or your test file (depending on what you actually want the page to have) and resend!